src/opflow/interface/opflowregi.cpp

```diff
@@ -134,13 +136,13 @@ extern PetscErrorCode OPFLOWSolverCreate_IPOPT(OPFLOW);
 #endif
 #if defined(EXAGO_ENABLE_HIOP)
 extern PetscErrorCode OPFLOWSolverCreate_HIOP(OPFLOW);
+#endif
```

Comment:
`#ifdef` stil wrong

Reply:
Okay I checked the source files and my comments stand.

@abhyshr and @wperkins can clarify here, but taking our sparse and sparse_gpu models here as an example.

If we build with sparse, we need to enable the sparse model. Then, within that conditional, we also need to check for RAJA to determine if we support sparse_gpu.

The key here is the conditionals are nested, whereas your changes has moved these conditionals to be sequential. Your changes modify behaviour

Reply:
Okay, catch your drift now. I've modified it to match the develop branch and will check it in if the test suite runs.